metadata/org.liquibase/liquibase-core/4.17.0/resource-config.json

```diff
@@ -14,6 +14,9 @@
           "typeReachable": "liquibase.util.LiquibaseUtil"
         },
         "pattern": "\\Qliquibase.build.properties\\E"
+      },
+      {
+        "pattern": "^www.liquibase.org/xml/ns/dbchangelog/dbchangelog-.*"
```

Comment:
This will include all the XSDs which are quite some. Is this really necessary?

Reply:
Besides that, it's missing a `condition`.

Reply:
Another option would be to only add `dbchangelog-latest.xsd` - then the users have to add their old Liquibase schemas on their own.